### DISCUSSION TITLE
可能在安卓上运行alas吗，比如termux之类的

### DISCUSSION THREAD

**Discussion by yuxinqi-chan** [LmeSzinc/AzurLaneAutoScript#2965] [General]
可能在安卓上运行alas服务吗，比如termux之类的，通过adb连接自己，这样云手机之类的也不一定要公网adb了。

**Comment by LmeSzinc** [comment]
看具体云手机厂商的潜规则，有的会直接卸载他们不希望你安装的应用，比如终端模拟类应用
